Content.Server/Station/StationSystem.cs

```diff
@@ -0,0 +1,168 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Maps;
+using Robust.Shared.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Log;
+
+namespace Content.Server.Station;
+
+/// <summary>
+/// System that manages the jobs available on a station, and maybe other things later.
+/// </summary>
+public class StationSystem : EntitySystem
+{
+    [Dependency] private GameTicker _gameTicker = default!;
+    private uint _idCounter = 1;
+
+    private Dictionary<StationId, StationInfoData> _stationInfo = new();
+    /// <summary>
+    /// List of stations for the current round.
+    /// </summary>
+    public IReadOnlyDictionary<StationId, StationInfoData> StationInfo => _stationInfo;
```

Comment:
why backing field?

Reply:
Makes sure that adding a station only goes through StationSystem.
Didn't see a reason to give other systems direct write access.